tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -2099,25 +2127,29 @@ static void
 editorOpenSwiftSourceInterface(StringRef Name, StringRef HeaderName,
                                ArrayRef<const char *> Args,
                                ResponseReceiver Rec) {
-  auto EditC = std::make_shared<SKEditorConsumer>(Rec,
-                                                  /*EnableSyntaxMap=*/true,
-                                                  /*EnableStructure=*/true,
-                                                  /*EnableDiagnostics=*/false,
-                                                  /*EnableSyntaxTree=*/false,
-                                                  /*SyntacticOnly=*/false);
+  auto EditC =
+      std::make_shared<SKEditorConsumer>(Rec,
+                                         /*EnableSyntaxMap=*/true,
+                                         /*EnableStructure=*/true,
+                                         /*EnableDiagnostics=*/false,
+                                         /*EnableSyntaxTree=*/false,
+                                         /*EnableSyntaxReuseInfo=*/false,
+                                         /*SyntacticOnly=*/false);
```

Comment:
Can we define a struct like `EditorConsumerOptions` to incorporate all these boolean values? Stacking them up like this can be error-prone.